command/command_test.go

```diff
@@ -0,0 +1,52 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestExecuteProgram(t *testing.T) {
+
+	test := ExternalCommand{
+		Path: "./test_command",
```

Comment:
Are you trying to execute a directory? The path to the binary is test_command/test_sample_program, right?